### PR TITLE
EVG-14207: evergreenize

### PR DESCRIPTION
**Pull request by hhoke**
autoship and smoke tests on linux, windows, and osx

### CODE REVIEW

**Comment by hhoke** [comment]
I tested out 8cffd26 modulo the version number, because I want the waterfall tests to work. (Once you do the autoship with a given version, all subsequent attempts will fail.) It worked.

File: evergreen.yml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      test_timeout: 15m
+    run_on:
+      - macos-1014
+    tasks:
+      - test-from-pypi
+
+  - name: windows
+    display_name: Windows
+    run_on:
+      - windows-64-vs2019-large
```
Comment (julianedwards) [inline evergreen.yml:116]: Should the smalls come first? Seems wasteful to grab a large here.

File: evergreen.yml [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+#######################################
+#           Buildvariants             #
+#######################################
+buildvariants:
+
+  - name: ubuntu2004
+    display_name: Ubuntu 20.04
+    run_on:
+      - ubuntu2004-small
```
Comment (julianedwards) [inline evergreen.yml:99]: Should we add more variants here and for the macos?

File: testdir/test_basic.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+'''
+to use, install curatorbin and then run this script.
```
Comment (julianedwards) [inline testdir/test_basic.py:3]: `To`

**Review by julianedwards — CHANGES_REQUESTED** [review]
Once we set up the inter-project dependency, this will update with the latest curator hash?

File: evergreen.yml (hunk 2 above)
Comment (hhoke) [reply evergreen.yml:99]: 1604 doesn't work and I don't want to tempt fate. I can add some more for the macos

**Comment by hhoke** [comment]
I think the best way to do that would be to trigger a special build based on publishing a tag in the curator repo, which I'm not sure how to do. There should be some way, but I expect I would have to spend a decent amount of time figuring that out and so left it for another ticket.

**Review by julianedwards — APPROVED** [review]

**Pull request merged**